suite-native/graph/src/hooks.ts

```diff
@@ -11,13 +13,26 @@ import { selectAccountByKey, selectMainnetAccounts } from '@suite-common/wallet-
 import { AccountKey } from '@suite-common/wallet-types';
 import { analytics, EventType } from '@suite-native/analytics';
 import { NetworkSymbol } from '@suite-common/wallet-config';
+import { atomWithUnecryptedStorage } from '@suite-native/storage';
 
 import { timeSwitchItems } from './components/TimeSwitch';
 
-type HoursValue = number | null;
+type TimeframeHoursValue = number | null;
+
+// Default is 720 hours (1 month).
+const DEFAULT_GRAPH_TIMEFRAME_HOURS = 720;
+
+const portfolioGraphTimeframeAtom = atomWithUnecryptedStorage<TimeframeHoursValue>(
+    'portfolioGraphTimeframe',
+    DEFAULT_GRAPH_TIMEFRAME_HOURS,
+);
+
+const accountToGraphTimeframeMapAtom = atomWithUnecryptedStorage<
```

Comment:
I am not sure about this, is it OK to save account key into unencrypted storage? It could leak also number of accounts, we will need to come with different solution.